License2Deploy/rolling_deploy.py

```diff
@@ -123,24 +124,19 @@ def get_lb(self):
   def calculate_autoscale_desired_instance_count(self, group_name, desired_state):
     ''' Search via specific autoscale group name to return modified desired instance count '''
     try:
-      cur_count = int(self.get_group_info(group_name)[0].desired_capacity)
       if desired_state == 'increase':
-        new_count = self.double_autoscale_instance_count(cur_count)
+        self.old_desired_capacity = int(self.get_group_info(group_name)[0].desired_capacity)
```

Comment:
this should moved back outside of the try block.
if you want to cache the value do an `if None:` set the value, the reason being is that if this method is called to`decrease` without being called to `increase` first it will currently set itself to `2` , correct?